scripts/utils/generate_ts.sh

```diff
@@ -62,6 +62,7 @@ for branch in $(git branch -r | grep origin/releases); do
 
   UNFLATTENED=false
 
+  # TODO: Remove this once all branches have been updated to not have the shared/ folder.
```

Comment:
Do we have a ticket for this? Let's create one if not and add the ticket number here please.